packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -330,14 +331,19 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
                 attached !== 'top' ? chatMyMessageStyles.bodyAttached : undefined,
                 mergeStyles(messageContainerStyle)
               ),
-              style: { ...createStyleFromV8Style(messageContainerStyle) }
+              style: { ...createStyleFromV8Style(messageContainerStyle) },
+              ref: messageRef
             }}
             root={{
               className: chatMyMessageStyles.root,
               onBlur: (e) => {
                 // copy behavior from North*
```

Comment:
Is it time to clear up the N* comments now?

Reply:
I prefer to have it there to add details why it's done in this way

Reply:
That's a good plan but really only tells future us what n* did rather than define the details of the tech decision making.

Reply:
the details are under this comment :) But I can delete the line about N*

Reply:
Updated!